content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/image.html

```diff
@@ -17,21 +17,26 @@
      data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
      data-sly-test="${image.src}"
      data-cmp-dmimage="${image.dmImage}"
+     data-cmp-smartcroprendition="${image.smartCropRendition}"
+     data-cmp-src="${image.srcUriTemplate ? image.srcUriTemplate : image.src}"
      data-asset-id="${image.uuid}"
      id="${image.id}"
      data-cmp-data-layer="${image.data.json}"
-     class="cmp-image${!wcmmode.disabled ? ' cq-dd-image' : ''}"
+     class="cmp-image${!wcmmode.disabled ? ' cq-dd-image' : ''}${image.autoSizesEnabled ? ' autosize' : ''}"
      itemscope itemtype="http://schema.org/ImageObject">
     <a data-sly-unwrap="${!image.imageLink.valid}"
        class="cmp-image__link"
        data-sly-attribute="${image.imageLink.htmlAttributes}"
        data-cmp-clickable="${image.data ? true : false}">
-        <img srcset="${image.srcset}" src="${image.src}"
-             loading="${image.lazyEnabled ? 'lazy' : ''}"
-             class="cmp-image__image"
-             itemprop="contentUrl"
-             width="${image.width}" height="${image.height}"
-             alt="${image.alt || true}" title="${image.displayPopupTitle && image.title}"/>
+        <noscript data-sly-unwrap="${!image.autoSizesEnabled}">
```

Comment:
Here is a codepen that proves the above statement https://codepen.io/buuhuu/pen/OJvoRpy

Reply:
The reason to put the `<img>` tag in a `<noscript>` tag is to prevent the browser from start loading the wrong rendition of the image. If the JS for `autoSizes` kicks in the browser start loading the image based on the calculated `sizes` attribute.

Reply:
I got that, still layout shift contributes to web performance metrics the same way as loading to large images does. 

Unfortunately an `<img>` without a `src` is not valid html. Setting a data uri src may cause issues on with redundant bytes being shipped and performance degradation on mobile devices due to decoding base64. So the only thing I could imaging is using a) a very low resolution version of the image or b) a transparent pixel that could even be prefetched. 

Both would eventually load unnecessary data. With lazy loading enabled the browser may abort the low resolution request when the javascript executes before the browser actually fetches the image.

If we would accept invalid html, we could avoid the eventual unnecessary roundtrip by setting not setting the `src`.

Styling the `<noscript>` the same way the `<img>` behaves does not work either unfortunately.